src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/WorkflowRunTests.kt

```diff
@@ -28,4 +33,28 @@ class WorkflowRunTests : IntegrationTest()
         val page = Jsoup.parse(response.text)
         Assertions.assertThat(page.selectFirst("#runWorkflowTabsVueApp")).isNotNull()
     }
+
+    @Test
+    fun `lists workflows`()
+    {
+        val sessionCookie = webRequestHelper.webLoginWithMontagu(runReportsPerm)
+
+        val name = "Interim report"
+        val key = "adventurous_aardvark"
+        val email = "test.user@example.com"
+        val date = Instant.now()
+
+        val repo = OrderlyWebWorkflowRunRepository()
+        repo.addWorkflowRun(WorkflowRun(name, key, email, date, emptyList(), emptyMap()))
+
+        val response = webRequestHelper.requestWithSessionCookie("/workflows", sessionCookie, ContentTypes.json)
```

Comment:
I understand that you have added the params to the model. I think integration test should test the endpoint as a whole `/workflows/?email=test.user@example.com&name=Interim%20report ` instead.

Reply:
Done!